cypress/page/index.ts

```diff
@@ -0,0 +1,17 @@
+import { SignInPage } from "./login.page";
+import { MenuContentPage } from "./menu-content.page";
+import { PaymentPage } from "./payment-step.page";
+import { ProductsListPage } from "./products-list.page";
+import { ShippingPage } from "./shipping-step.page";
+import { AddressPage } from "./address-step.page";
+import {ShopingCartPage} from "./shoping-cart.page"
+
+export {
+    MenuContentPage,
+    ProductsListPage,
+    SignInPage,
+    AddressPage,
+    ShippingPage,
+    PaymentPage,
+    ShopingCartPage, 
+};
```

Comment:
this should not be required since each export is done at the end of each page object class